### ISSUE TITLE
documentation missing for multiple features

### ISSUE THREAD

**Issue by Simon-Laux** [prebuild/node-gyp-build#32]
- allow injecting a prebuild using an env var
- support prebuild.test for test script setup

**Comment by mafintosh** [comment]
You wanna send a PR?

**Comment by Simon-Laux** [comment]
nope, as I only really understood the 3rd point (preinstall and postinstall arguments).
Maybe when I get some more time in the future to learn the other 2 features to a point where I can explain/document them.
